test/functional/wallet_importmulti.py

```diff
@@ -304,10 +308,9 @@ def run_test(self):
                               False,
                               error_code=-8,
                               error_message='Internal must be set to true for nonstandard scriptPubKey imports.')
-        address_assert = self.nodes[1].getaddressinfo(address)
-        assert_equal(address_assert['iswatchonly'], False)
-        assert_equal(address_assert['ismine'], False)
-        assert_equal('timestamp' in address_assert, False)
```

Comment:
n.b. we've lost this test now, but that seems fine

Reply:
I didn't think this test was important, so I removed it, but I've now readded it.